opensoundscape/localization.py

```diff
@@ -256,3 +697,100 @@ def localize(
             return u0
         else:
             return u1
+
+
+def gillette_localize(
+    receivers=list,
+    delays=list,
+    temp=20,
+    m=[0],
+    exact=True,
+    summary=False,
+    confint=False,
+    alpha=0.05,
+    td_error=False,
+    total_td_error=False,
+):
+    """
+    Calculate the estimated location of a sound's source using the
+    algorithm laid out in Gillette and Silverman (2008)
```

Comment:
give an overview of the algorithm's approach: how does it formulate the problem as a least squares problem?